File: javatoh/Step12/src/Disk.java
import java.util.*;

public class Disk {

	private int size;
	
	Disk(int size){
		this.size = size;
		
	}
	
	// it is returning its size.
	public int getSize() {
		return this.size;
	}
	
	public void draw() {
		
		for(int i =0; i<this.size; i++) {
			System.out.print("X");	
		}
		System.out.println("");
		
	}
	
}

File: pythontoh/Step9/disk.py
from colorama import init
from colorama import Fore, Back, Style

init()
class Disk:
    def __init__(self, size, color):
        self.size = size
        self.color = color

    def get_size(self):
        return self.size
        
    def draw(self):
        print('                      ', end='')   # put some filler
        for x in range(1, self.size, 1):
            # by specifying end we are saying don't go the next line after print.
            print(self.color + 'X' , end='')
        # move the current cursor to the next line, so that our base will draw properly
        print(Fore.WHITE +'')   # put WHITE back so that it will reset to WHITE
